webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -295,15 +295,15 @@
 		<role-name>IbisObserver</role-name>
 	</security-role>
 	<security-role>
-		<description>can browse the error queue and resend or delete the messages in it, reload configurations, start and stop adapters, has all IbisObserver permissions too</description>
+		<description>can browse the error queue and resend or delete the messages in it, reload configurations, start and stop adapters, has all IbisObserver permissions</description>
 		<role-name>IbisDataAdmin</role-name>
 	</security-role>
 	<security-role>
-		<description>can do a full reload and test a pipeline, has all IbisDataAdmin permissions too</description>
+		<description>can do a full reload and has all IbisDataAdmin permissions</description>
 		<role-name>IbisAdmin</role-name>
 	</security-role>
 	<security-role>
-		<description>can execute jdbc query, send jms message and test a service, has all IbisAdmin and IbisWebService permissions too</description>
+		<description>can execute jdbc query, send jms message and test a service, has all IbisAdmin and IbisWebService permissions</description>
```

Comment:
Ik denk dat we de hele overlap uit deze rollen moeten halen (en dan niet alleen in deze beschrijvingen)

Reply:
Tja, deze rol definities zijn sowieso een beetje obscure. Ik zeg altijd, roep `/iaf/api?allowedRoles=true` aan, dan zie je het zelf.